x-pack/test/feature-aware/build.gradle

```diff
@@ -0,0 +1,20 @@
+import org.elasticsearch.gradle.precommit.PrecommitTasks
+
+apply plugin: 'elasticsearch.build'
+
+dependencies {
+  compile 'org.ow2.asm:asm:6.2'
+  compile "org.elasticsearch:elasticsearch:${version}"
+  compile "org.elasticsearch.plugin:x-pack-core:${version}"
+  testCompile "org.elasticsearch.test:framework:${version}"
+}
+
+forbiddenApisMain.enabled = true
+forbiddenApisMain {
+  signaturesURLs = [PrecommitTasks.getResource('/forbidden/jdk-signatures.txt')] // does not depend on core, only jdk signatures
```

Comment:
We have core on the classpath so maybe we can drop this?

Reply:
Ah, yeah. That's leftover from when my first implementation was based on strings (names of the classes) rather than now where I took a dependency to be able to refer to the classes directly.